ui/base-tool/src/main/res/drawable/ic_tract_live_share.xml

```diff
@@ -0,0 +1,35 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
this should have been left in the `tract-tool`, is there a reason you copied it to `base-tool`?

Reply:
It is used in the `tool_option_settings_sheet.xml`.  I did remove the duplicate from the tract-renderer.

Reply:
instead of just putting all possible options in base-tool, we probably should refactor to allow each actual tool type to define additional options to display.

Maybe a horizontal RecyclerView that is provided items that can be rendered into the appropriate buttons. That would allow a Tract tool to provide the live-share action without you needing to put the resources for it in base-tool